app/src/main/java/com/hamdan/forzenbook/view/login/MainLogin.kt

```diff
@@ -101,22 +101,20 @@ private fun UserInputtingContent(
         imeAction = ImeAction.Next, onNext = { focusManager.moveFocus(FocusDirection.Down) }
     )
     if (emailError && email != "") {
-        Text(text = "Invalid email or email length longer than 30 characters")
+        Text(text = R.string.login_email_error.toString())
     }
     Spacer(modifier = Modifier.height(PaddingValues.medPad_2))
     SubmitButton(
         onSubmission = onSubmission,
-        label = "Login",
+        label = R.string.login_button_text.toString(),
```

Comment:
??? No lol you need to use stringResource() not just try and toString it

Reply:
I remembered some transformation needed to happen on it and thought it was just toString() xd. Changed all cases to stringResource()